lib/src/bloc_hook.dart

```diff
@@ -15,8 +16,8 @@ T useCubit<T extends BlocBase>({
   final cubit = useMemoized(() => _injectorFn<T>(), keys);
 
   useEffect(() {
-    onInit?.call(cubit);
-    return cubit.close;
+    DisposeCommand? dispose =  onInit?.call(cubit);
```

Comment:
With this change cubit will never be closed if user did't do it manually in onInit method which is optional